src/MicrosoftTeams.schema.json

```diff
@@ -35,17 +35,20 @@
                 "websiteUrl": {
                     "type": "string",
                     "description": "The url to the page that provides support information for the app.",
-                    "maxLength": 2048
+                    "maxLength": 2048,
+                    "pattern": "^[Hh][Tt][Tt][Pp][Ss]*://"
                 },
                 "privacyUrl": {
                     "type": "string",
                     "description": "The url to the page that provides privacy information for the app.",
-                    "maxLength": 2048
+                    "maxLength": 2048,
+                    "pattern": "^[Hh][Tt][Tt][Pp][Ss]://"
```

Comment:
Use "pattern": "^[Hh][Tt][Tt][Pp][Ss]?://" since this does't have to be an https URL